### PR TITLE
fix `tests` 

### PR DESCRIPTION
**Pull request by Dillon-Roller**

### CODE REVIEW

File: src/pieces.rs [hunk 1]
```diff
@@ -65,15 +74,17 @@ impl AddAssign<Move> for Position {
 // Add unit tests at the bottom of each file. Each tests module should only have access to super (non integration)
 #[cfg(test)]
 mod tests {
-    use super::Position;
+    use super::*;
```
Comment (DanielAlbl) [inline src/pieces.rs:74]: Nice, so this imports all of our structs basically?

File: src/moves.rs [hunk 2]
```diff
@@ -4,6 +4,15 @@ pub struct Move {
     pub dy: i8,
 }
 
+impl Move {
```
Comment (DanielAlbl) [inline src/moves.rs:7]: Can this go in the `impl` below?

File: src/pieces.rs [hunk 3]
```diff
@@ -65,15 +74,17 @@ impl AddAssign<Move> for Position {
...
 
     #[test]
     fn test_add_position() {
-        let p = Position { x: 1, y: 1 };
-        assert_eq!(p + p, Position { x: 2, y: 2 });
+        let mut p = Position::new(0, 0);
```
Comment (DanielAlbl) [inline src/pieces.rs:78]: Is new better than `Position { 2, 2 }`?

File: src/pieces.rs (hunk 1 above)
Comment (Dillon-Roller) [reply src/pieces.rs:74]: `super` in this case would be `piece` and so includes everything in the `piece` module

File: src/moves.rs (hunk 2 above)
Comment (Dillon-Roller) [reply src/moves.rs:7]: thx missed that one

File: src/pieces.rs (hunk 3 above)
Comment (Dillon-Roller) [reply src/pieces.rs:78]: true

**Review by DanielAlbl — APPROVED** [review]

**Pull request merged**